docs/contributor/arch/010-logging-flow-healthiness-status-api.md

```diff
@@ -0,0 +1,62 @@
+# 10. Log Flow Health Status API
+
+Date: 2024-03-28
+
+## Status
+
+Proposed
+
+## Context: Key Events in the Fluent Bit Log Flow
+
+[ADR 003: Integrate Prometheus With Telemetry Manager Using Alerting](003-integrate-prometheus-with-telemetry-manager-using-alerting.md) describes a concept for self-monitoring and [ADR 008 Telemetry Flow Health Status API](008-telemetry-flow-healthiness-status-api.md) defines the related pipeline conditions that are derived from the OpenTelemetry Collector metrics. This ADR focuses on events in the Fluent Bit log flow.
+
+![Fluent Bit Data Flow](../assets/fluent-bit-data-flow.drawio.svg "Fluent Bit Data Flow")
+
+### Log Rotation
+
```

Comment:
For which purpose are we using a bullet list in the following subsections? Aren't these just normal sentences, forming a normal paragraph?

Reply:
Makes sense, removed them.